assets/airports/kbos.json

```diff
@@ -398,316 +528,8185 @@
+                "RIFLE": ["RIFLE", "KYSKY", "CUTOX", "CUJKE", "FERNZ", ["LONEY", "A110+"], "RODEE"]
+            },
+            "body": [["EURRO", "A100+|S270"], ["OOSHN", "A90+|A140-|S250"]],
             "rwy": {
-                "4R":  [["TTERI", "A100|S250"], "WAATR", ["FLUTI", "A90"], "GRIFI", ["GGABE", "A60|S220"], ["JOBEE", "A60|S210"]],
-                "4L":  [["TTERI", "A100|S250"], "WAATR", ["FLUTI", "A90"], "GRIFI", ["GGABE", "A60|S220"], ["JOBEE", "A60|S210"]],
+                "4R": [["TTERI", "A90+|S250"], "WAATR", ["FLUTI", "A90+"], "GRIFI", ["GGABE", "A60|S220"], ["JOBEE", "A60|S210"]],
+                "4L": [["TTERI", "A90+|S250"], "WAATR", ["FLUTI", "A90+"], "GRIFI", ["GGABE", "A60|S220"], ["JOBEE", "A60|S210"]],
                 "15R": [["GRGIO", "A90-"], ["BRGIT", "A70"], ["HNOVR", "A50|S220"], ["RDHOK", "A50|S210"], ["TKMAN", "A50|S210"]],
                 "22R": [["GRGIO", "A90-"], ["BRGIT", "A70"], ["ADDDA", "A60|S220"], ["LEEZI", "A60|S210"]],
                 "22L": [["GRGIO", "A90-"], ["BRGIT", "A70"], ["ADDDA", "A60|S220"], ["LEEZI", "A60|S210"]],
-                "27":  ["PUDJJ", ["AYBEE", "A60|S210"]],
-                "33L": [["TTERI", "A100|S250"], "WAATR", "PLGRM", ["BECHH", "A50|S220"], ["SCITU", "A50|S210"]],
-                "32":  [["TTERI", "A100|S250"], "WAATR", "PLGRM", ["BECHH", "A50|S220"], ["SCITU", "A50|S210"]]
-            }
+                "27": ["PUDJJ", ["AYBEE", "A60|S210"]],
+                "33L": [["TTERI", "A90+|S250"], "WAATR", "PLGRM", ["BECHH", "A50|S220"], ["SCITU", "A50|S210"]],
+                "32": [["TTERI", "A90+|S250"], "WAATR", "PLGRM", ["BECHH", "A50|S220"], ["SCITU", "A50|S210"]]
+            },
+            "draw": [
+                ["RIFLE", "KYSKY", "CUTOX", "CUJKE", "FERNZ", "LONEY", "RODEE", "EURRO"],
+                ["ENE", "NIKKO", "EURRO", "OOSHN", "TTERI", "WAATR", "FLUTI", "GRIFI"],
+                ["MERIT", "FOSTY", "IYANA", "KLEBB", "SOBAH", "LONEY"],
+                ["AJJAY", "SUFFI", "NIKKO"],
+                ["FEXXX", "FOSTY"],
+                ["PROVI", "IYANA"],
+                ["OOSHN", "GRGIO", "BRGIT", "HNOVR", "RDHOK", "TKMAN"],
+                ["BRGIT", "ADDDA", "LEEZI"],
+                ["WAATR", "PLGRM", "BECHH"],
+                ["OOSHN", "PUDJJ", "AYBEE"]
+            ]
         },
         "ROBUC2": {
             "icao": "ROBUC2",
             "name": "Roebook Two",
             "entryPoints": {
                 "RUIZE": [],
-                "JFK":   ["JFK", "NEWES", ["FEXXX", "A230"]],
+                "JFK": ["JFK", "NEWES", ["FEXXX", "A230"]],
                 "MERIT": ["MERIT"]
             },
-            "body": [["RUIZE", "A230"], ["AWLIN", "A200"], ["BANKI", "A170"], ["ROBUC", "A130"], ["PROVI", "A110"], "JOODY"],
+            "body": [["RUIZE", "A230-|A210+"], ["AWLIN", "A170+"], ["BANKI", "A160+"], ["ROBUC", "A120+|A190-|S260"], ["PROVI", "A110+"], "JOODY"],
```

Comment:
Since we already have one change to make on this PR, might as well reorder the `RUIZE` restriction in minimum-to-maximum order: `["RUIZE", "A210+|A230-"]`